src/Tizen.NUI/src/public/Utility/GraphicsTypeExtensions.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// The Dp Extension class is density independent pixel(dp) unit coverter for float object.
+    /// One dp is a virtual pixel unit that's roughly equal to one pixel
+    /// on a medium-density(160dpi) screen.
+    /// To convert Dp to pixel, use Pixel property or,
+    /// See <see cref="Tizen.NUI.GraphicsTypeManager" /> and <see cref="Tizen.NUI.GraphicsTypeConverter" /> also.
+    /// </summary>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public static class GraphicsTypeExtensions
+    {
+        /// <summary>
+        /// Converter float pixel to dp.
+        /// 100.0f.ToDp() = 100.0f in 160dpi display.
+        /// </summary>
+        /// <param name="pixel">The float pixel unit value to be converted dp unit.</param>
+        /// <returns>The float dp unit value.</returns>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static float ToDp(this float pixel)
+        {
+            if (GraphicsTypeManager.Instance.TypeConverter is DpTypeConverter)
+            {
+                return GraphicsTypeManager.Instance.ConvertFromPixel(pixel);
+            }
+            else
+            {
+                return DpTypeConverter.Instance.ConvertFromPixel(pixel);
+            }
+        }
+
+        /// <summary>
+        /// Converter float dp to pixel.
+        /// 100.0f.ToPixel() = 100.0f in 160dpi display.
+        /// </summary>
+        /// <param name="dp">The float dp unit value to be converted pixel unit.</param>
+        /// <returns>The float pixel unit value.</returns>
+        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static float ToPixel(this float dp)
+        {
+            if (GraphicsTypeManager.Instance.TypeConverter is DpTypeConverter)
+            {
+                return GraphicsTypeManager.Instance.ConvertToPixel(dp);
+            }
+            else
+            {
+                return DpTypeConverter.Instance.ConvertToPixel(dp);
+            }
```

Comment:
이렇게 분기하여 변환하는것은 목적에 맞지 않은것 같습니다.

`GraphicsTypeManager`의 Instance.TypeConverter를 사용하는 이유는 추후에 변경될 수 있는 변환 방법에 대해
기존 코드 변경 없이 그 변환을 적용하기 위함으로 보입니다.

그래서 변환 방법이 변경되더라도 `GraphicsTypeManager.Instance.ConvertToPixel`을 호출하는 코드는 변경이 필요 없게 하기 위함입니다.

그런데 이렇게 분기하여 호출하는것은 그 의도를 반영하지 못하는것입니다.
즉, 어떤 변환이 되었든간에 약속된 변환식(User Scale이 적용되었든 추가적인 Scale을 거치든..)을 거쳐 Pixel로 변환하는것이 목적이지, 단순하게 DP *  DPI / 160으로 변환하는것이 목적이 아니기 때문입니다.

그리고 `GraphicsTypeManager.Instance.ConvertToPixel`이 계산값은 XAML에서 dp로 입력된 값을 Pixel로 변환하는데 사용하는데, 코드에서 10.ToDP()했을때, 사용되는 변환 함수가 달라지면 어디서 변환한지에 따라 다른 결과값이 발생하므로 동일한 변환 함수를 사용하도록 수정되어야 합니다.
```suggestion
            return GraphicsTypeManager.Instance.ConvertToPixel(dp);
```

Reply:
일단 의도했던 목적은,
추후 기본 제공하는 Converter가 Dp가 아닌 다른 Converter로 확장될 경우를 가정한 내용이긴 한데요.
1. 사용자가 DpTypeConverter를 상속하여 만든 Custom Class를 Manager의 Converter로 설정한 경우 1번에 의해 처리가 되고
2. 사용자가 DpTypeConverter가 아닌 다른 TypeConverter를 만들어서 Manager에 설정한 경우 해당 Converter를 사용하지 않고 Default DpTypeConverter를 사용하기 위함입니다.

Reply:
일단 지금 그런 고민을 하는게 좀 불필요한 고민인거 같아서 말씀해주신 것처럼 TypeManager의 Convert 기능을 적용하도록 수정했습니다.